superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -289,6 +301,17 @@ export default function DataSourcePanel({
     [lists.columns, showAllColumns],
   );
 
+  const showInfoboxCheck = () => {
```

Comment:
this can follow the same process as datasourceTypeCheck

Reply:
I tried adjusting this and learned that it won't work, I need `sessionStorage.getItem('showInfobox')` to run each time this renders to check if the user has closed the infobox for this session.

Reply:
oh ok, that makes sense!

Reply:
same comment as below.. I would name this something that sounds like a boolean, and you an also rewrite it to be a constant value rather than a function: 
```
const somethingThatSoundsLIkeABoolean = !(sessionStorage.getItem('showInfobox') === 'false')
```

Reply:
if the truthy value is "true", then maybe: 
`const somethingThatSoundsLIkeABoolean = sessionStorage.getItem('showInfobox') === 'true'